R/load_files.R

```diff
@@ -142,6 +142,7 @@ read_in_manifest_file <- function(manifest_file, available_formats) {
           e
         ))
       })
+      data.table::setDT(df)
     } else if (manifest_ext %in% c("text/tsv",
```

Comment:
In this branch `data.table::setDT(df)` is also needed, as `utils::read.table` returns data.frame :(
There is also no test for that (scenario: manifest is tsv file)

Reply:
Done. Replaced `read.table` and `read.csc` by `data.table::fread`

Reply:
✔️